frontend/src/app/coworking/widgets/room-dialogue/room-dialogue.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+
+@Component({
+  selector: 'room-dialog',
+  template: `
+    <div class="dialog-container">
+      <button
+        class="dialog-close-button"
+        mat-icon-button
+        (click)="onCloseClick()">
+        <mat-icon>close</mat-icon>
+      </button>
+      <h1 class="dialog-title" mat-dialog-title>Room Information</h1>
+      <div class="dialog-content" mat-dialog-content>
+        <p>Room: {{ data.id }}</p>
+        <p>Capacity: {{ data.capacity }}</p>
+        <p>Description: {{ data.description }}</p>
```

Comment:
Might be a good idea to either provide a placeholder, or hide this entirely, if a description for the room is blank

Reply:
Looping in @aarjavjain2002